run.py

```diff
@@ -0,0 +1,45 @@
+from mpi4py import MPI
+import numpy as np
+import adios2
+
+import src.convert_stl as convert_stl
+import src.embed_stl as embed_stl
+
+def run(stl_file):
+
+    comm = MPI.COMM_WORLD
+    rank = comm.Get_rank()
+    size = comm.Get_size()
```

Comment:
Is this still needed? if it's not implemented yet I'd just remove it and the import